opennms-doc/guide-admin/src/asciidoc/text/operation/cassandra21x.adoc

```diff
@@ -0,0 +1,112 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+=== Cassandra monitoring
+
+This section describes some of the metrics _OpenNMS_ collects for monitoring an _Apache Cassandra_ database
+infrastructure. The category `Cassandra21x` is used to enable the data collection on a node. Also, the service
+name `Cassandra21x` is used to bind this data collection to a node's interface. The _JMXCollector_ is used
+to retrieve the _MBean_ entities from the _Cassandra_ node.
+
+==== Thread pools
+
+_Apache Cassandra_ is based on a so called _Staged Event Driven Architecture_ (SEDA). This seperates different
+operations in stages and these stages are loosely coupled using a messaging service. Each of these components use
+queues and thread pools to group and execute their tasks. _OpenNMS_ collects the data for the following thread pools:
+
+* AntiEntropyStage
+* CacheCleanupExecutor
+* CommitLogArchiver
+* CompactionExecutor
+* GossipStage
+* HintedHandoff
+* InternalResponseStage
+* MemtableFlushWriter
+* MemtablePostFlush
+* MemtableReclaimMemory
+* MigrationStage
+* MiscStage
+* PendingRangeCalculator
+* Sampler
+* ValidationExecutor
+* CounterMutationStage
+* MutationStage
+* ReadRepairStage
+* ReadStage
+* RequestResponseStage
+* Native-Transport-Requests
+
+For each of these the following metrics are collected by _OpenNMS_:
+
+[options="header, autowidth"]
+|===
+| Name | Description
+| `ActiveTasks` | Tasks that are currently running
+| `CompletedTasks` | Tasks that have been completed
+| `CurrentlyBlockedTasks` | Tasks that have been blocked due to a full queue. This value should be 0 on healthy clusters.
+| `PendingTasks` | Tasks queued for execution. Of course, low values (<15) are healthy.
+| `TotalBlockedTasks` | Number of tasks that have been blocked in total
+|===
+
```

Comment:
Sometimes there is a . at the end of the description, sometimes there is not. Can we make it the same?